CHANGELOG.md

```diff
@@ -1,6 +1,21 @@
 # CHANGELOG
 
 -----
+## `v3.2.0-beta`
```

Comment:
My bad. I totally forgot about those. I can see there are lot of small commits. Lets talk offline.